Makefile

```diff
@@ -0,0 +1,49 @@
+.PHONY: help fetch update cleanup dist publish
+SHELL=/bin/bash
+GEMSPEC=$(shell ls *.gemspec | head -1)
+VERSION=$(shell ruby -e 'puts Gem::Specification::load("$(GEMSPEC)").version')
+PROJECT=$(shell ruby -e 'puts Gem::Specification.load("$(GEMSPEC)").name')
+GEM=$(PROJECT)-$(VERSION).gem
+
+help: ## Print this message
+	@awk 'BEGIN { FS = ":.*##"; print "Usage:  make <target>\n\nTargets:" } \
+		/^[-_[:alpha:]]+:.?*##/ { printf "  %-15s%s\n", $$1, $$2 }' $(MAKEFILE_LIST)
+
+fetch: ## Grab latest from theme repo
+	mkdir -p tmp
+	make cleanup
+	curl -o tmp/assets.zip https://codeload.github.com/MITLibraries/mitlib-style/zip/master
+	unzip -o -d tmp tmp/assets.zip
+
+update: ## Synchronize latest (run fetch_assets first)
```

Comment:
Can you move the cleanup actions down to the `cleanup` target and have this depend on it, like above? Also, it looks like maybe your intent is that the `fetch` target always gets run before this one? If so, make it depend on `fetch` so you don't have to type `make` twice.